common/djangoapps/enrollment/data.py

```diff
@@ -260,10 +262,11 @@ def get_course_enrollment_info(course_id, include_expired=False):
         CourseNotFoundError
 
     """
-    course_key = CourseKey.from_string(course_id)
-    course = modulestore().get_course(course_key)
-    if course is None:
+    try:
+        course_key = CourseKey.from_string(course_id)
+        course = CourseOverview.get_from_id(course_key)
+        return CourseField().to_native(course, include_expired=include_expired)
+    except CourseOverview.DoesNotExist:
```

Comment:
Nit: it's usually better to just wrap the operation that might fail in the `try: ... except: ...` clause. In this case, I think that's `CourseOverview.get_from_id`. You can use an `else:` block to wrap the code that should only happen if there's no exception.